### PR TITLE
Update the README.md file to describe the repo.

### PR DESCRIPTION
**Pull request by jagjennings**
Edited the README.md file to reflect my portfolio.

### CODE REVIEW

File: portfolio/src/main/webapp/ceramics.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <title>Jag's Ceramic Creations</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="content">
+      <h1>Some Wheel-Thrown Pieces</h1>
+      <div class="image-thumbnails">
+        <a href="images/stanley-1.jpg"><img src="/images/ceramics.jpg"/></a>
```
Comment (qihuang2) [inline portfolio/src/main/webapp/ceramics.html]: did you still want the stanley-1.jpg image? I don't think you have it in your images file

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -13,16 +13,16 @@
...
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['I like to make wheel-thrown pottery.', 'I love watching Survivor.', 'I am learning Mandarin.', 'I was born on Friday the 13th.','I am a Libra.'];
```
Comment (qihuang2) [inline portfolio/src/main/webapp/script.js]: nit: add space in between last 2 items in array

File: portfolio/src/main/webapp/ceramics.html (hunk 1 above)
Comment (jagjennings) [reply portfolio/src/main/webapp/ceramics.html]: deleted the stanley image.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (jagjennings) [reply portfolio/src/main/webapp/script.js]: added a space.

**Review by qihuang2 — APPROVED** [review]

**Pull request merged**